polaris/tasks/ocean/seamount/viz.py

```diff
@@ -0,0 +1,126 @@
+import cmocean  # noqa: F401
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import xarray as xr
+from mpas_tools.ocean.viz.transect import compute_transect, plot_transect
+
+from polaris import Step
+from polaris.mpas import cell_mask_to_edge_mask
+from polaris.viz import plot_horiz_field
+
+
+class Viz(Step):
+    """
+    A step for plotting the results of the default seamount forward step
+    """
+
+    def __init__(self, component, indir):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        component : polaris.Component
+            The component the step belongs to
+
+        indir : str
+            the directory the step is in, to which ``name`` will be appended
+        """
+        super().__init__(component=component, name='viz', indir=indir)
+        self.add_input_file(
+            filename='mesh.nc', target='../../init/culled_mesh.nc'
+        )
+        self.add_input_file(filename='init.nc', target='../../init/init.nc')
+        self.add_input_file(
+            filename='output.nc', target='../forward/output.nc'
+        )
+
+    def run(self):
+        """
+        Run this step of the task
+        """
+        ds_mesh = xr.load_dataset('mesh.nc')
+        ds_init = xr.load_dataset('init.nc')
+        ds = xr.load_dataset('output.nc')
+
+        x_min = ds_mesh.xVertex.min().values
+        x_max = ds_mesh.xVertex.max().values
+        y_mid = ds_mesh.yCell.median().values
+
+        x = xr.DataArray(data=np.linspace(x_min, x_max, 2), dims=('nPoints',))
```

Comment:
```suggestion
        x = xr.DataArray(data=[x_min, x_max], dims=('nPoints',))
```
Maybe this is clearer?

Reply:
Done, thanks.